docs/articles/commands/intro.md

```diff
@@ -11,8 +11,7 @@ Right now, your bot is console-mute. Let's change it. Let's make it output all i
 To do that, add the following options to your `DiscordConfiguration`:
 
 ```cs
-UseInternalLogHandler = true,
-LogLevel = LogLevel.Debug
+MinimumLogLevel = Microsoft.Extensions.Logging.LogLevel.Debug
```

Comment:
Or... `using Microsoft.Extensions.Logging`.

Reply:
I thought of that, but i also want to convey that this was Microsofts enum not Dsharpplus.  If ur comfortable will just simplify it to the using, i can do that

Reply:
That's a fair point, can be left up to discussion.

Reply:
Considering DSharpPlus no longer has a LogLevel enum, I think it's fine just to simplify it.

Reply:
While on the topic of `using Microsoft.Extensions.Logging`, you could also throw in/mention there is custom logging support.

Reply:
@Neuheit that is also a fair point!  I was doing a little thinking and if we simplify it and turn `DiscordConfiguration` into a link to the api documentation, it should satisfy both types of thinking.  Do you guys agree?

Reply:
Regarding custom logging, #605 might be a starting point. It includes some basic information on how what works, and, IIRC, how to get a custom one going. Also might be worth mentioning that one can also plug implementations such as Serilog, NLog, Apache log4net, etc.

Reply:
Ya, when I write the Custom Logging page, I'm gonna provide examples from Serilog as that is what I'm most comfortable with personally and my hatred of log4net lol

Reply:
> my hatred of log4net lol

Don't worry, hating Apache software is normal.